### DISCUSSION TITLE
Update Frequency on Microsoft Store

### DISCUSSION THREAD

**Discussion by Jonibhoni** [mifi/lossless-cut#1244] [General]
Is there a reason that the paid version on Microsoft Store is apparently no more or only very rarely updated? It's still version 3.39.0.

**Comment by mifi** [comment]
Hi. this is explained here: https://github.com/mifi/lossless-cut#difference-between-app-stores-and-github-download

Basically I want the Store version to be as perfect as can be (because Store version is not possible for people to downgrade.) So I want to make sure there are no big issues before releasing to the Store. Up until 3.46.2 was released (5 days ago), there were some pretty serious unresolved bugs, that I wanted to fix those first, like UI lagginess #1097. However as of 3.46.2, I don't think there are any more critical bugs, so I plan on releasing 3.46.2 to the stores very soon 🚀

As for frequency, there is no fixed release schedule/frequency for the stores, it depends on the complexity of features/changes added.

**Reply by Jonibhoni** [reply]
Ahhhh... sorry I missed it, it's right there, right on the start page! Well, now subsequent visitors will have the answer here as well... :sweat_smile:

Anyway, that's very cool to hear that this is actually on purpose for the benefit of stability!

I love what you're doing! Keep up the great work! :raised_hands:
